CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+## Welcome!
+
+We're so glad you're thinking about contributing to an 18F open source project! If you're unsure about anything, just ask -- or submit the issue or pull request anyway. The worst that can happen is you'll be politely asked to change something. We love all friendly contributions.
```

Comment:
That said, we should probably update the ['canonical' source](https://github.com/18F/open-source-policy) for these files.